CMakeLists.txt

```diff
@@ -770,7 +765,7 @@ ENDIF ()
 OPTION (LinAlgBackend "Set global linear algebra backend library for all modules")
 IF (LinAlgBackend STREQUAL "EIGEN3")
 	FIND_PACKAGE(Eigen3 ${EIGEN_VERSION_MINIMUM} REQUIRED)
```

Comment:
doesn't make much sense anymore, because previous it has been either found or bundled.